hs_core/templates/pages/genericresource.html

```diff
@@ -1053,7 +1054,7 @@ <h4>Contributors</h4>
                                 </div>
                             {% endif %}
                             <!-- ========= RESOURCE SPECIFIC TAB ========= -->
-                            {% if site or extended_metadata_exists or res_add_metadata or metadata_form %}
+                           <!--  {% if site or extended_metadata_exists or res_add_metadata or metadata_form %}
```

Comment:
@philtuskha And what is being hidden here?

Reply:
THis? 
![image](https://user-images.githubusercontent.com/21337693/38748992-08af3256-3f1e-11e8-937b-a0e81085b372.png)

I frankly don't know what this is or what it is used for.  It didn't seem relevant to MYHPOM, however.  So I hid it.

I sluethed around a little more and figured out why this tab started to appear without any content and I think I have a less intrusive change.    

I'll push the change to the branch next.